ebuild-maintenance/package-and-slot-moves/text.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/maintenance-tasks/">
+<chapter>
+<title>Package and slot moves</title>
+
+<body>
+<p>
+This chapter describes the use of package and slot moves. The package update
+mechanism is a powerful too, and needs to be used with caution. In particular,
+the following must be noted:
```

Comment:
```suggestion
the followings must be noted:
```

Reply:
Wait, 'following' has a plural? oO

Reply:
> Wait, 'following' has a plural? oO

My dictionary says that it has a plural only for the meaning "group of followers", which doesn't apply here.